src/api/searchApi.ts

```diff
@@ -64,14 +64,17 @@ export async function groupSearch(searchQuery: GQLQuerySearchArgs, context: Cont
   return json.map((result: IGroupSearchResult) => ({
     ...result,
     resources: result.results.map((contentTypeResult) => {
-      const path =
-        contentTypeResult.contexts.find((c) => (subjects.length === 1 ? c.rootId === subjects[0] : c.isPrimary))
-          ?.path ?? contentTypeResult.paths?.[0];
+      const searchCtx = contentTypeResult.contexts.find((c) =>
+        subjects.length === 1 ? c.rootId === subjects[0] : c.isPrimary,
+      );
+      const path = searchCtx?.path ?? contentTypeResult.paths?.[0];
+      const url = searchCtx?.url ?? contentTypeResult.paths?.[0];
```

Comment:
Kan dette gjøres i tax?

Reply:
Nei, for dette er via search-api, så her må vi gjøre litt prosessering basert på søkeparameter.